docs/source/essentials/00-queries.mdx

```diff
@@ -32,28 +31,32 @@ query HeroQuery($id: String!) {
 }
 ```
 
-Here, `query` is the operation type and `HeroQuery` is the operation name.
-Apollo-android will generate a `HeroQuery` class that you can construct (with variables) and pass to `ApolloClient#query(query)`:
+If we define this query in its own `.graphql` file, Apollo Android will generate a corresponding `HeroQuery` class that we can then use to [execute the query](#executing).
 
-```kotlin
-val response = apolloClient.query(feedQuery)
-```
+To generate classes, Apollo Android requires your server's schema and a set of `.graphql` files (one for each operation in your application). It uses these to generate code you can use to execute queries and access typed results.
```

Comment:
Does it actually have to be one operation per file? Or can you define multiple operations in a single file?

Reply:
It can be multiple operations in a single file. In the end, all the `.graphql` files are concatenated together.